chain/client/src/client.rs

```diff
@@ -2178,8 +2178,10 @@ impl Client {
                     )
                 });
 
-            debug!(target: "catchup", ?me, ?sync_hash, progress_per_shard = ?format_shard_sync_phase_per_shard(&new_shard_sync, use_colour), "Catchup");
+            // For colour decorators to work, they need to printed directly. Otherwise the decorators get escaped, garble output and don't add colours.
```

Comment:
Do you mean that it only works with println! (and friends) but not with the tracing library?

Reply:
`progress_per_shard` is a `HashMap`, and when it gets printed using its derived `Debug` implementation, both keys and values are formatted nicely, but the derived implementation escapes the special characters in the values because it needs to print the whole container.
Printing via `println!` has the same effect on the formatting as via `tracing::info!`.